test/metabase/api/notify_test.clj

```diff
@@ -71,19 +70,26 @@
                                      (merge {:synchronous? true}
                                             payload)))))]
       (testing "sync just table when table is provided"
```

Comment:
Changed these names to reflect the actual function called. Neither one will trigger a full sync.